src/js/player.js

```diff
@@ -2481,23 +2481,14 @@ class Player extends Component {
 }
 
 /*
- * Global player list
- *
- * @type {Object}
- */
-Player.players = {};
```

Comment:
That's how it's been since `videojs.options` was introduced so it must not be that easy. All of our docs examples show videojs.options as an existing object, so I'm not sure why you would assume you need to overwrite the whole object.

If we change that then we'd be adding a 4th set of options that get merged in when the player is built (init options, tag options, videojs.options, prototype.options_). That seems more annoying for debugging purposes.